src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -296,7 +315,7 @@ class Crystal::Doc::Generator
   end
 
   def doc(obj : Type | Method | Macro | Constant)
-    doc = obj.doc
+    doc = obj.doc.try &.strip.lchop(":showdoc:").strip
```

Comment:
If we're going with this change (which I believe is right thing to do), I guess we should also strip the `:nodoc:` comments as well.

Reply:
Why? Comments starting with `:nodoc:` won't show up anywhere.

Reply:
Indeed! Seems like I'm having a mind-haze today 🤦‍♂️